planetscale/imports.go

```diff
@@ -0,0 +1,306 @@
+package planetscale
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/pkg/errors"
+)
+
+type DataImportSource struct {
+	HostName            string `json:"hostname"`
+	Database            string `json:"schema_name"`
```

Comment:
The controller in the API uses this name because the PlanetScale downstream database name is passed as `database`, to avoid confusion.
``` ruby
  database: connection[:schema_name],
  user: connection[:username],
```